desc/compute/_metric.py

```diff
@@ -1823,3 +1825,79 @@ def _gradtheta(params, transforms, profiles, data, **kwargs):
 def _gradzeta(params, transforms, profiles, data, **kwargs):
     data["|grad(zeta)|"] = jnp.sqrt(data["g^zz"])
     return data
+
+
+@register_compute_fun(
+    name="gbdrift",
+    # Exact definition of the magnetic drifts taken from
+    # eqn. 48 of Introduction to Quasisymmetry by Landreman
+    # https://tinyurl.com/54udvaa4
+    label="\\mathrm{gradB-drift} = 1/B^{2} * (\\bm{b}\\times\\nabla (B)) \\dot"
```

Comment:
`\\bm` doesn't work in matplotlib, need to use `\\mathbf`
also `\\dot` puts an overdot, i think you want `\\cdot`